web/src/components/pages/login.test.js

```diff
@@ -0,0 +1,23 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import ConnectedLogin from '../../containers/login';
+import { getStore } from '../../tests/testUtils';
+
+test('Login button is disabled without input values', () => {
+  const initialState = {
+    app: {
+      authed: false,
+      username: null,
+      errormessage: null,
+      loginDisabled: true,
+    };
```

Comment:
I think this should be a comma, not a semicolon.

Reply:
Good catch. Hope I got it all right this time 🤞